src/openrct2-ui/windows/Guest.cpp

```diff
@@ -759,10 +757,10 @@ void window_guest_viewport_init(rct_window* w){
 
     window_event_invalidate_call(w);
 
-    w->viewport_focus_coordinates.x = focus.coordinate.x;
-    w->viewport_focus_coordinates.y = focus.coordinate.y;
-    w->viewport_focus_coordinates.z = focus.coordinate.z;
-    w->viewport_focus_coordinates.rotation = focus.coordinate.rotation;
+    w->viewport_focus_coordinates.x = coordinate.x;
+    w->viewport_focus_coordinates.y = coordinate.y;
```

Comment:
This would not pass the sprite_type information which is also stored in this memory address.